providers/amazon/tests/unit/amazon/aws/operators/test_datasync.py

```diff
@@ -441,9 +443,11 @@ def test_init_fails(self, mock_get_conn):
         with pytest.raises(AirflowException):
             self.set_up_operator(source_location_uri=None)
         with pytest.raises(AirflowException):
-            self.set_up_operator(destination_location_uri=None)
+            self.set_up_operator(destination_location_uri=None, task_id="datasync_task2")
```

Comment:
And here.  Assuming this case is failing because of duplicate task id, this doesn't belong in provider tests.